bcache/get_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bcache
+
+import (
+	"testing"
+	"math"
+)
+
+func TestParseStats(t *testing.T) {
+	// test dehumanize
+	dehumanizeTests := []struct {
+		in  []byte
+		out float64
+	}{
+		{
+			in:  []byte("542k"),
```

Comment:
You'll want to add some nonsensical test cases to check when errors occur as well.